tests/test_api.py

```diff
@@ -507,23 +507,6 @@ def test_is_target_in_pathpattern(self):
             )
 
 
-    def test_delegation_class(self):
-        # empty keys and roles
-        delegations_dict = {"keys":{}, "roles":[]}
-        delegations = Delegations.from_dict(delegations_dict.copy())
-        self.assertEqual(delegations_dict, delegations.to_dict())
-
-        # Test some basic missing or broken input
-        invalid_delegations_dicts = [
-            {},
-            {"keys":None, "roles":None},
```

Comment:
Don't we care about this case anymore? Is it covered somewhere else?

Reply:
Testing that delegations are `{"keys":None, "roles":None}` can't be done with the current code and I don't see a reason to test for it, because in JSON there is no such thing as `None` and we are calling `case_dict = json.loads(test_case_data)` before making the actual tests which throws the `json.decoder.JSONDecodeError` exception.